conf/messages

```diff
@@ -20,12 +20,16 @@ transferAgreement.english=The records are all in English
 transferAgreement.digital=The records are all Digital
 transferAgreement.droAppraisalSelection=I confirm that the DRO has signed off on the appraisal and selection decision.
 transferAgreement.droSensitivity=I confirm that the DRO has signed off on the sensitivity review and that all records are OPEN i.e. no FOI exemptions apply to these records.
-upload.header=Upload
-upload.title=Upload
+upload.header=Upload Records
+upload.title=Upload Records
+upload.message=Select a folder
+upload.details=Which file extensions are not permitted?
+upload.summary=Cannot upload the following file extensions: .zip, thumbs.db, .exe
```

Comment:
I might give the last three slightly more descriptive names so it's more obvious what they do when you read the template code. Maybe:

- upload.selectFolder
- upload.fileExtensionTitle
- upload.fileExtensionSummary

Reply:
I've made the changes that you suggested, not sure why I didn't keep it consistent/more clear before  🤦‍♀